### PR TITLE
completed react-components

### PR DESCRIPTION
**Pull request by t-doan**
react-components

### CODE REVIEW

File: react-components/my-app/src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      <h1>Vite + React</h1>
+      <div className="card">
+        <button onClick={() => setCount((count) => count + 1)}>
+          count is {count}
+        </button>
+        <p>
+          Edit <code>src/App.tsx</code> and save to test HMR
+        </p>
+      </div>
+      {CustomButton()}
```
Comment (RobertGardner) [inline react-components/my-app/src/App.tsx:29]: This is not the correct way to call React components. It works, but it's not idiomatic. Use `<>` syntax.

File: react-components/my-app/src/CustomButton.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export default function CustomButton() {
```
Comment (RobertGardner) [inline react-components/my-app/src/CustomButton.tsx:1]: Don't use default exports in your LFZ code.

File: react-components/my-app/src/CustomButton.tsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  return (
+    <button value="Click me!" type="button">
```
Comment (RobertGardner) [inline react-components/my-app/src/CustomButton.tsx:3]: What is the purpose of "value" here?

**Review by RobertGardner — CHANGES_REQUESTED** [review]
Fix the call.

File: react-components/my-app/src/App.tsx [hunk 4]
```diff
@@ -26,7 +26,7 @@ function App() {
           Edit <code>src/App.tsx</code> and save to test HMR
         </p>
       </div>
-      {CustomButton()}
+      <CustomButton />
```
Comment (t-doan) [inline react-components/my-app/src/App.tsx:29]: fixed the import and call for the component

**Review by RobertGardner — APPROVED** [review]

**Pull request merged**